### DISCUSSION TITLE
REPL input does not work on ESP8266 NodeMCU v3

### DISCUSSION THREAD

**Discussion by ahenyao** [micropython/micropython#12588] [ESP8266]
Hi,
I got my esp8266 nodemcu v3 board and I wanted to install micropython because I know it. I installed it using Thonny, and I can't input to REPL using Putty.

**Comment by robert-hh** [comment]
Did you disable flow control at Putty?

**Comment by ahenyao** [comment]
I don't know where is it, so i suppose no

**Comment by robert-hh** [comment]
It is in the settings->connection->serial. There is a config item for Flow Control. verify as well, that you use the right COM port.

**Comment by ahenyao** [comment]
when I click "open" nothing happens. tried with both 9600 and 115200
![image](https://github.com/micropython/micropython/assets/91829037/a77e2c60-25b0-4c48-bece-79ea68b7f913)

**Reply by robert-hh** [reply]
The speed should be 115200. After pushing `open`, the cursor should be in a blank screen. What happens if you then push reset at the ESP8266? There should some weird characters pop up, and at the end the MicroPytoon prompt. The weird characters are normal for ESP8266, since the boot messages are sent at 74880 baud.

**Comment by ahenyao** [comment]
this happens when i push reset 
![image](https://github.com/micropython/micropython/assets/91829037/7f3210cc-9002-4427-8102-a0249c7277af)
opened serial from putty home screen and then this
![image](https://github.com/micropython/micropython/assets/91829037/084932c4-c4ce-48bb-a0b5-9ddc7163427b)

**Reply by robert-hh** [reply]
Looks better. So it was a different COM port.

**Comment by ahenyao** [comment]
just plugged to another usb port ¯\_(ツ)_/¯
